build/build.rb

```diff
@@ -89,6 +89,10 @@ def verify_run(output)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{git_checkout.commit_sha}.#{FILE_TYPE[imgfac_target]}"
     destination = DESTINATION_DIRECTORY.join(file_name)
     $log.info `mv  #{source} #{destination}`
+    $log.info "Generating image checksums"
```

Comment:
My opinion is to have 2 separate PRs.  One to add the functionality in the same format as the rest of the script.  Once that is merged, another PR to optimize the whole script with the suggestions being discussed.

@Fryguy Please chime in with your thoughts.